File: Morning Snack/Java/FactorialOf.java
import java.util.Scanner;
public class FactorialOf{
    public static void main(String[]args){
    Scanner input = new Scanner(System.in);

    System.out.print("Enter a number: ");
    int userInput = input.nextInt();
    int result;
    int factorial = 1;
    int number;

    for(number = 1; number <= userInput; number++){
        result = number;
        factorial *= result;        
    }

    System.out.println(factorial);  


    }
}

File: Morning Snack/JavaScript/FactorialOf.js
const input = require('prompt-sync')();
let userInput = input('Enter number: ');

let result;
let factorial = 1;
let number;

    for(number = 1; number <= userInput; number++){
        result = number;
        factorial *= result;        
    }

    console.log(factorial);  

